views/js/tools/screen/tester.js

```diff
@@ -0,0 +1,177 @@
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2018 (original work) Open Assessment Technologies SA ;
+ */
+/**
+ * @author Jean-Sébastien Conan <jean-sebastien@taotesting.com>
+ */
+define([
+    'lodash',
+    'i18n',
+    'core/format',
+    'taoClientDiagnostic/tools/getConfig',
+    'taoClientDiagnostic/tools/getLabels',
+    'taoClientDiagnostic/tools/getStatus'
+], function (_, __, format, getConfig, getLabels, getStatus) {
+    'use strict';
+
+    /**
+     * Some default values
+     * @type {Object}
+     * @private
+     */
+    var _defaults = {
+        id: 'screen',
+        threshold: {
+            width: 1024,
+            height: 768
+        }
+    };
+
+    /**
+     * Placeholder variables for custom messages
+     * @type {Object}
+     * @private
+     */
+    var _placeHolders = {
+        WIDTH: '%WIDTH%',
+        HEIGHT: '%HEIGHT%'
+    };
+
+    /**
+     * A list of thresholds
+     * @type {Array}
+     * @private
+     */
+    var _thresholds = [{
+        threshold: 0,
+        message: __('The device screen does not meet the minimum resolution of %s!'),
+        type: 'error'
+    }, {
+        threshold: 33,
+        message: __('The device screen has the minimum required resolution.'),
+        type: 'success'
```

Comment:
Make sense to set warning status for the threshold value. Total rating is has warning style in this case:
![image](https://user-images.githubusercontent.com/11025793/35849745-ae93011c-0b33-11e8-8b90-982b65c8a558.png)

Reply:
Your are right!